README.md

```diff
@@ -10,59 +10,60 @@ Run an Ethereum compatible ~~parachain~~ (and blockchain for now, until parachai
 ## Install (linux)
 
 ### Get the code
-
+Get the tutorial specific tag of the PureStake/Moonbeam repo:
 ```bash
-git clone -b moonbeam-tutorials https://github.com/PureStake/moonbeam
+git clone -b tutorial-v2 https://github.com/PureStake/moonbeam
 cd moonbeam
 ```
 
-### Rust developer environment
+### Setting up enviroment
 
 Install Substrate pre-requisites (including Rust):  
 ```bash
 curl https://getsubstrate.io -sSf | bash -s -- --fast
 ```
 
-## Build Parachain
+Run the initialization script, which checks the correct rust nightly version and adds the WASM to that specific version:
+```bash
+./scripts/init.sh
+```
+
+## Build Standalone
+Build the corresponding binary file:
+
+```bash
+cd node/standalone
+cargo build --release
+```  
 
-Build Wasm and native code:  
+## Build Parachain
+Build the corresponding binary file:
 ```bash
 cargo build --release
 ```  
-(Building for the first time will take a long time, to install and compile all the libraries)
+The first build takes a long time, as it install and compiles all the necessary libraries.
```

Comment:
```suggestion
The first build takes a long time, as it compiles many dependencies.
```

"Install" isn't really the right word here. It is compiling that code to be part of our binary.

Reply:
Modified it thanks 👍